assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -67,6 +67,10 @@
 	--count-border-color: #{$gray};
 	--count-color: #{$gray-text-min};
 	--profile-gravatar-user-secondary-info-color: #{$gray-darken-20};
+
+	--form-field-active-background: #{$blue-medium};
```

Comment:
can we just use $blue-wordpress for the active-background and remove the need for these variables entirely? It would change legacy but make everything simpler.

Reply:
cc @drw158 @shaunandrews

Reply:
I'd be fine with that.